pkg/translate/translate_value.go

```diff
@@ -138,15 +138,27 @@ func (t *ReverseTranslator) initK8SMapping(valueTree map[string]interface{}) err
 
 // NewReverseTranslator creates a new ReverseTranslator for minorVersion and returns a ptr to it.
 func NewReverseTranslator(minorVersion version.MinorVersion) (*ReverseTranslator, error) {
+	return newReverseTranslator(minorVersion, maxFallbackNum)
+}
+
+func newReverseTranslator(minorVersion version.MinorVersion, fallbackNum uint) (*ReverseTranslator, error) {
 	t := ReverseTranslators[minorVersion]
 	if t == nil {
+		if fallbackNum > 0 && minorVersion.Minor > 0 {
```

Comment:
Use the function here.

Reply:
I tried to create a common fallback function, but it seems making things more complex and even harder to read. Can we first merge this into release-1.4 we can address readabilities in the next round of PR.